<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmAddWord</class>
 <widget class="QWidget" name="frmAddWord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="windowIcon">
   <iconset resource="resfile.qrc">
    <normaloff>:/new/icons/icons/addw.png</normaloff>:/new/icons/icons/addw.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_11">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lblQuality">
         <property name="text">
          <string>quality</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbUpdQual">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbAWquality"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>class</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbUpdType">
         <property name="text">
          <string>update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbAWtype"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="cbDrvKey">
           <property name="text">
            <string>deriv key</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtDrvKey">
           <property name="text">
            <string>0</string>
           </property>
           <property name="maxLength">
            <number>6</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>crum</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtAWcrum">
           <property name="minimumSize">
            <size>
             <width>61</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>61</width>
             <height>27</height>
            </size>
           </property>
           <property name="inputMask">
            <string>D00; </string>
           </property>
           <property name="maxLength">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbAWA">
           <property name="text">
            <string notr="true">A</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbAWB">
           <property name="text">
            <string notr="true">B</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbUpdCrum">
           <property name="text">
            <string>update</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QToolButton" name="btTemplTr">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>tr:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="btTemplIntrQ">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>intr (qual):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="btTemplAdvb">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>with advb</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QToolButton" name="btTemplIntr">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>intr:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="btTemplRefl">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>(refl)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="btTemplPrep">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>with prep</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>coptic  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbUpdCopt">
         <property name="text">
          <string>update</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLineEdit" name="txtAWcopt">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="maxLength">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="txtAWcoptTr">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Coptic</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QToolButton" name="btEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>edit</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>czech</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbUpdCz">
         <property name="text">
          <string>update</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="txtAWcz">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>31</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="maxLength">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>english</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbUpdEn">
         <property name="text">
          <string>update</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="txtAWeng">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>31</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="maxLength">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>greek</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbUpdGr">
         <property name="text">
          <string>update</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="txtAWgr">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="maxLength">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="txtAWgrTr">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>31</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="btAWsave">
       <property name="text">
        <string>save +</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>action</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QRadioButton" name="rbClose">
          <property name="text">
           <string>close</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbNewW">
          <property name="text">
           <string>new word</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbNewD">
          <property name="text">
           <string>new derivation</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>578</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <tabstops>
  <tabstop>cbUpdCopt</tabstop>
  <tabstop>cbUpdCz</tabstop>
  <tabstop>cbUpdEn</tabstop>
  <tabstop>txtAWeng</tabstop>
  <tabstop>cbUpdGr</tabstop>
  <tabstop>txtAWgr</tabstop>
  <tabstop>btAWsave</tabstop>
  <tabstop>rbClose</tabstop>
  <tabstop>rbNewD</tabstop>
  <tabstop>rbNewW</tabstop>
 </tabstops>
 <resources>
  <include location="resfile.qrc"/>
 </resources>
 <connections/>
</ui>
